framework/db/Connection.php

```diff
@@ -618,9 +625,10 @@ protected function initConnection()
      */
     public function createCommand($sql = null, $params = [])
     {
-        $command = new Command([
+        $command = \Yii::createObject([
```

Comment:
`Instance::ensure` is used in cases created object should match some base class or interface. Since we have none for the `Command` it does not necessary here.
Also, it will search a component inside DI container (application), which is unlikely suitable logic here.
I suppose it can be left as it is now.